benchmarks/bench_kernel_pca_solvers_time_vs_n_components.py

```diff
@@ -118,6 +119,19 @@
         r_time[j, i] = time.perf_counter() - start_time
         # check that the result is still correct despite the approximation
         assert_array_almost_equal(np.abs(r_pred), np.abs(ref_pred))
+    
+    # D- randomized_value
+    print("  - randomized_value solver")
```

Comment:
Could you please paste the resulting plot image of the two benchmarks into the PR conversation ?